app.py

```diff
@@ -45,10 +45,13 @@ def inner_decorator(func):
         @wraps(func)
         def wrapped_func(*args, **kwargs):
             token = None
-            if request.method == 'GET':
+            token = request.cookies.get('token')
```

Comment:
Remove the line `token = None`. If there is no cookie named `token` then the variable `token` will be `None` anyways